tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -113,7 +115,7 @@ if [ $DEVICE_TYPE = "gpu" ]; then
     if [[ $BUILD_EXTR_PAR = *--enable_training* ]]; then
       ${PYTHON_EXE} -m pip install -r ${0/%install_deps.sh/training\/requirements.txt}
 
-      if [[ $BUILD_EXTR_PAR = *--enable_training_python_frontend_e2e_tests* || $BUILD_EXTR_PAR = *enable_training_pipeline_e2e_tests* ]]; then
+      if [[ $BUILD_EXTR_PAR = *--enable_training_python_frontend_e2e_tests* || $BUILD_EXTR_PAR = *enable_training_pipeline_e2e_tests* || $DISTRIBUTED = true ]]; then
```

Comment:
can we replace parsing of BUILD_EXTR_PAR with this distributed option?

Reply:
It can.
My only concern is that I am not completely sure of all the scenarios where these ```BUILD_EXTR_PAR``` are currently being passed (which all pipelines make use of this). So, I don't want to be creating problems in other pipelines.
So far, I know they are being passed by the nightly pipeline under the changes from #5702 . But I think we can separate that part out for another pull request. Please let me know your thoughts on this.

Reply:
yes, i suppose BUILD_EXTR_PAR might be parsed for other values. but for this particular case, searching in tools/ci_build/github, i think those e2e test options are only used here:
tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-e2e-test-nightly-pipeline.yml
tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend-test-ci-pipeline.yml

it would be great to clean it up. this is a good step. i'm fine with doing that in another PR too.

Reply:
Let me get this in another PR since I think there are a few more steps involved in that process (like cleaning up these variables in ```run_dockerbuid.sh``` as well.